cmd/arcstat/arcstat

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env python3
 #
 # Print out ZFS ARC Statistics exported via kstat(1)
-# For a definition of fields, or usage, use arctstat.pl -v
+# For a definition of fields, or usage, use ./arctstat -v
```

Comment:
This looks like a typo arc**t**stat should be arcstat? Also, drop the "./".

Reply:
@rlaager nice catch, fixed